src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java

```diff
@@ -66,4 +67,38 @@ public void waitForLoadingSpinnerToFinishReloadingPage() {
       waitForElementPresenceByBy(By.cssSelector(".loading-overlay.hidden"));
     }
   }
+
+  //TODO: Remove this and use combination od logUrl
```

Comment:
when creating TODO please create a ticket and link ticket number so it won't be there forever.